src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -485,3 +485,77 @@ public static DisplayInformation GetForMonitor(IntPtr monitor)
     }
 #endif
 }
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+#if UAC_VERSION_1
+#if NET
+    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.10240.0")]
+#endif
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class DataTransferManagerInterop
+    {
+        private static readonly global::System.Guid IDataTransferManagerInterop_IID = new global::System.Guid(0x3A3DCD6C, 0x3EAB, 0x43DC, 0xBC, 0xDE, 0x45, 0x67, 0x1C, 0xE8, 0x00, 0xC8);
+
+        private static readonly global::System.Guid riid = new global::System.Guid(0xA5CAEE9B, 0x8708, 0x49D1, 0x8D, 0x36, 0x67, 0xD2, 0x5A, 0x8D, 0xA0, 0x0C);
+
+#if NET
+        private static global::WinRT.IObjectReference objectReference => global::WinRT.ActivationFactory.Get<global::WinRT.IObjectReference>("Windows.ApplicationModel.DataTransfer.DataTransferManager", IDataTransferManagerInterop_IID);

```

Comment:
CI error D:\a\_work\1\s\src\Projections\Windows\Generated Files\net6.0\CsWinRT\ComInteropHelpers.cs(516,110): error CS0308: The non-generic method 'ActivationFactory.Get(string)' cannot be used with type arguments

```suggestion
        private static global::WinRT.IObjectReference objectReference => global::WinRT.ActivationFactory.Get("Windows.ApplicationModel.DataTransfer.DataTransferManager", IDataTransferManagerInterop_IID);
```